docs/guide/acquiring_data/database.rst

```diff
@@ -267,7 +267,7 @@ sections.
    ...        self.instrument = instrument
    ...        self.wave = MockObject(wavemin=wavelength, wavemax=wavelength, waveunit='nm')
    >>> vso.VSOClient().search = Mock(return_value=[MockQR(f"2011050800000{t}", 'AIA', w) for
-   ...                                   t,w in zip([0,0,2,3], [17.1, 21.1, 9.4, 33.5])])
+   ...                                   t,w in zip([0,0,2,3], [17.1, 21.1, 9.4, 33.5])]) # doctest: +REMOTE_DATA
```

Comment:
why? this should be setting this up for offline doc tests.

Reply:
`VSOClient` cannot be instantiated without internet access, that's why I had to add `+REMOTE_DATA`. What would be the best way to handle this? The easiest way I can think of is to make `search` a `staticmethod` in `VSOClient`.

Reply:
I can mock it or I can temporarily override the constructor of `VSOClient` using a helper function. I'll see which approach is more elegant.